### DISCUSSION TITLE
I see "In progress"

### DISCUSSION THREAD

**Discussion by SimTheGreat** [OpenMined/courses#236] [Project: Privacy Product Specification] [answered]
hi!
I submitted my project about 3 weeks ago and I am not sure if I did it correctly.
I see "In progress" and at the bottom I see 
"Thank you for your submission! Out mentors will review your work and give you feedback within 1-2 days. You will receive a notification and an email when your project has been reviewed. In the meantime, see what others did by discussing this project on our Discussion Board."
Is it ok?

**Comment by yemikifouly** [comment, marked as answer]
Hello Samriddhi! 👋 You need not worry. Those messages means that everything was submitted successfully and that once reviews will start you will get feedback on your project. 🙂
